src/components/molecules/TableComponent/TableComponent.tsx

```diff
@@ -81,6 +81,34 @@ const TableComponent: React.FunctionComponent<TableComponentPropsType> = ({
 
   const itemClasses = classNames("TableComponent__item", itemStyles);
 
+  const renderedUserPictures = useMemo(
+    () =>
+      users &&
+      users.length >= 0 &&
+      users.map((user) => {
+        const { src: profilePic, onError: onLoadError } = determineAvatar({
+          user,
+        });
+
+        return (
+          <img
+            onClick={() => openUserProfileModal(user.id)}
+            key={user.id}
+            className="TableComponent__profile-icon"
+            src={profilePic}
+            onError={onLoadError}
+            title={(!user.anonMode && user.partyName) || DEFAULT_PARTY_NAME}
```

Comment:
`anonMode` was removed previousle (original comment in `image.ts`). Is that one back on the table now?

```
// checking user.anonMode ruled against by https://github.com/sparkletown/internal-sparkle-issues/issues/1615#issuecomment-991054012
```

Reply:
removed `anonMode` check here: https://github.com/sparkletown/sparkle/pull/2760/commits/db98df9b57d0e55acca8872b685e36307a96aa59